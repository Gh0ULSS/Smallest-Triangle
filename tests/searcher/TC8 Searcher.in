29.15, -46.9
33.77, -18.25
-48.8, -43.03
0.22, 13.16
32.14, 31.71
-39.74, 36.38
-40.42, 31.96
-3.96, 8.09
25.95, -29.15
-43.78, 38.52
-32.77, -21.51
-41.57, -10.56
-20.61, -2.04
44.35, -0.59
-7.22, 26.16
18.85, -29.65
34.43, 29.31
-24.28, 2.36
14.83, -13.4
17.53, 11.32
-6.47, 27.75
-33.53, 20.4
28.07, 16.84
40.16, 30.2
-4.85, -44.47
-39.82, -48.59
-32.04, 28.22
41.6, 32.24
41.34, 3.85
31.58, 14.69
47.17, -44.6
30.43, -16.92
-25.97, 30.56
1.32, 27.2
-17.08, -12.14
21.87, 17.32
2.17, 3.09
22.21, 21.47
20.28, 27.37
-46.11, 43.36
-19.87, 33.04
-17.65, 26.76
-24.46, -13.6
-27.92, -21.02
-19.26, -6.55
-16.46, -7.38
38.8, -30.86
-42.59, 46.73
-47.11, -22.2
-44.72, 27.32
-8.54, 47.81
-24.84, 7.39
-27.69, 9.19
18.41, -36.62
-44.66, 5.43
-10.21, -28.67
7.57, 26.07
25.12, -13.49
-10.55, 42.37
-45.67, -39.7
-6.09, 49.88
24.33, 2.28
-13.69, -24.07
-33.04, -25.1
30.92, 18.37
-42.21, -22.4
-35.24, -21.28
9.47, 5.1
-37.21, -24.96
-0.19, -34.63
24.04, -37.73
49.95, -25.44
2.32, 20.41
-10.36, 10.23
-35.14, -0.81
-8.21, -15.66
24.26, 46.04
-33.25, -34.54
-14.02, -8.34
20.82, 37.69
-42.47, -0.6
-13.4, -9.86
34.06, -23.63
-25.86, -27.76
1.89, 36.94
-32.91, -36.71
4.23, 28.02
47.67, 7.28
-12.29, -21.86
37.17, -35.88
24.6, -34.73
-4.8, 13.57
-16.05, 38.11
0.35, 26.47
2.03, -12.84
40.66, -37.42
-46.95, -11.05
31.77, 35.49
30.56, -9.17
17.28, 27.87
37.39, 46.14
47.15, 19.81
32.06, -34.66
35.03, -19.23
-4.02, -48.49
-2.54, -3.24
-34.77, 41.22
43.57, 26.95
-43.92, -39.18
49.91, -20.97
-35.38, 33.03
-42.38, 9.71
20.7, -15.4
6.61, -25.68
17.23, -13.57
41.75, -42.71
34.95, -2.53
5.69, -45.87
-8.0, -5.05
-2.04, 44.88
-0.34, -18.71
-9.37, 16.9
34.4, -10.52
-12.4, -17.04
35.22, -46.37
-23.31, 31.34
-32.35, 5.27
24.92, -2.98
-0.12, -5.71
48.6, 8.15
-21.79, 34.56
14.44, 18.74
49.6, -45.1
-35.94, 49.37
-48.31, -21.27
9.03, -36.61
-14.21, -48.47
36.53, -3.21
-25.78, 34.02
-1.75, 40.71
-48.97, 10.17
10.29, 8.88
-25.54, 12.08
20.11, -20.53
27.36, 36.38
-11.71, 41.09
18.82, -11.27
42.17, -30.25
11.38, -31.73
-8.67, 6.33
-32.98, 25.16
-49.64, -10.16
-11.65, 23.41
-9.94, 48.29
-32.37, 8.39
-48.31, 26.66
-32.77, 40.21
10.0, 14.78
-37.0, 36.02
8.72, -49.96
-49.2, 28.37
-1.27, 37.97
-33.84, 43.92
-22.52, -16.21
-16.1, -15.91
28.59, 35.86
-10.67, 33.93
20.69, -28.15
3.53, -34.01
-27.13, -25.22
-2.93, 26.9
17.04, 49.56
-33.22, 37.11
-17.31, 7.46
-12.11, -11.78
48.07, 48.42
-26.33, -23.56
-7.32, -43.18
-39.71, 7.62
-15.24, 22.66
1.62, -44.13
41.37, 24.84
-31.54, -39.32
-28.11, -33.62
-3.28, -8.33
-38.73, 10.26
9.88, -14.28
40.56, 21.6
-38.33, -19.39
8.29, -16.18
40.18, 28.59
46.0, 48.01
-3.17, -11.91
25.13, 44.1
3.36, -29.49
-14.81, 26.84
8.72, 12.82
27.44, -37.11
-30.14, -27.36
-9.14, -32.49
7.25, 35.76
-44.36, 16.72
0.1, 18.17
-42.04, -13.72
-8.02, -27.81
-47.53, 20.38
-21.45, -3.66
19.69, 44.68
-26.94, 22.03
27.57, 37.52
-43.55, -2.21
-44.68, -36.55
19.2, -45.13
8.67, 10.45
-34.29, 8.98
27.32, 42.86
32.61, -25.09
18.67, 24.33
-33.01, 10.57
-38.88, 46.95
-30.91, -2.07
-16.47, 18.17
-25.98, 17.02
44.48, 22.02
-46.69, 44.52
47.42, -10.1
21.42, -6.87
-42.84, 34.93
20.67, 21.09
-19.76, -45.66
-25.93, -14.59
-21.2, 12.89
-43.41, 48.9
31.2, 22.9
-38.96, -17.31
-24.03, 9.49
19.98, 36.15
12.57, 22.47
5.73, -22.88
17.91, 13.07
-46.41, 27.6
-17.11, 39.34
-17.1, 20.14
-22.47, -2.38
-12.93, -18.6
-14.24, 25.27
38.67, 5.49
-25.69, -13.57
43.32, -32.18
3.81, 32.61
21.12, 44.36
10.35, 43.8
-21.07, -38.93
47.95, -19.57
12.91, -9.4
44.83, 26.73
35.78, 48.37
45.01, -25.67
-18.93, 21.72
-37.35, 2.1
9.66, 48.65
-5.93, -27.31
-17.26, 43.89
-44.33, -31.22
-15.71, -8.65
-36.15, 18.4
-38.58, 18.82
-15.04, -22.03
21.03, -37.35
14.17, -21.95
-22.2, 4.97
-48.85, -47.42
13.41, 24.7
-25.22, 18.97
-26.2, 29.18
-22.14, 39.01
-10.74, 6.84
-30.07, -48.27
12.65, 5.2
11.6, 49.25
32.42, 20.35
34.35, 10.52
28.27, 49.62
13.24, 41.41
17.16, 23.04
-45.3, -20.76
49.19, 28.95
-44.45, 32.13
49.79, -47.74
39.65, 27.25
-15.6, -32.48
2.18, -24.77
-20.18, 30.91
28.2, -0.23
-41.05, 26.84
48.57, -27.1
1.75, -47.08
10.76, 28.14
-11.52, 14.49
-9.06, -0.13
-20.84, -9.97
9.68, -6.15
38.87, -38.1
-39.76, 14.13
-33.57, -0.83
-26.97, 47.14
14.18, 16.56
42.6, -3.6
-12.2, -30.9
-39.0, 20.1
-30.6, 24.58
-31.17, 1.46
30.66, 3.3
10.46, 19.15
37.54, -42.25
-46.06, 18.44
-1.88, -8.02
-36.27, -41.91
-17.14, -19.85
-24.17, 44.87
-8.33, -34.95
-29.85, 3.84
-27.38, 17.19
22.12, -21.23
32.43, -14.03
-34.61, -47.49
-34.67, 49.5
3.42, -19.73
6.95, 10.19
-28.66, -1.64
-13.42, -26.36
47.73, 2.09
-48.77, 38.54
-17.03, 24.41
49.42, 44.01
8.2, 29.32
26.74, -21.39
34.84, -28.63
14.59, -10.88
-3.55, 15.18
42.18, 27.41
-30.32, 13.4
27.74, -17.7
-2.59, 29.14
0.23, 4.87
-17.67, 46.04
-23.53, -24.38
27.64, 11.71
8.19, 37.67
-5.02, -16.58
-28.74, -29.03
-18.72, 49.27
-35.64, 37.82
-29.22, 49.08
19.07, -3.35
48.82, -48.27
-33.09, -27.46
-20.52, -37.99
-40.03, -28.0
25.52, -1.92
2.68, 2.17
22.49, -5.95
19.66, -35.49
5.74, 23.02
-48.84, 3.31
-1.59, 31.36
-44.93, 17.96
-43.95, -21.74
-9.89, -26.11
-33.48, -1.9
-41.56, -41.44
-13.95, 30.09
11.72, 10.85
27.48, 46.42
24.78, -4.3
27.84, 9.8
2.75, -36.33
-0.72, -16.33
-13.59, 33.03
18.32, 47.57
-15.78, 5.73
28.5, 43.52
-49.15, 36.12
4.65, 22.63
4.58, 31.12
5.85, -39.02
16.77, 17.87
-21.92, -48.72
18.71, -31.22
-48.93, -8.42
-47.27, 43.64
-30.03, -23.38
12.22, 33.62
-20.44, -30.73
16.28, 10.68
43.43, -29.87
-16.51, -42.68
-37.17, 13.43
2.14, -31.97
-32.06, -28.49
-45.75, -19.74
22.76, -24.37
-21.62, -7.24
-2.11, -44.11
-39.38, 1.47
-27.65, 3.82
39.07, 20.11
-41.38, -28.5
33.02, 26.42
-2.52, 14.71
-0.18, -22.23
23.29, -44.03
-48.49, 40.27
38.14, 18.46
33.15, -34.02
5.11, 23.93
48.13, 4.18
-27.79, 48.08
-38.83, -29.25
-28.18, 23.07
47.79, -5.43
32.24, -3.47
-31.32, -37.98
19.97, 40.89
-24.22, -49.28
-36.51, -5.94
35.1, 47.27
44.12, -4.09
-40.28, 48.37
-18.65, 33.77
41.68, 5.78
18.66, 46.01
-0.91, 17.91
36.41, 43.15
38.42, 40.6
-46.88, 38.13
28.17, 39.77
-10.95, -46.08
31.56, 46.75
-31.72, 23.79
49.1, -40.64
-36.18, -38.09
-30.05, 0.14
39.8, -24.5
13.17, -17.16
44.0, 16.42
-17.13, 11.31
-14.63, -27.04
-29.86, -31.58
-42.57, 31.39
-31.17, -6.04
-17.43, 48.9
-24.88, -35.33
-20.39, 21.4
8.25, 49.2
47.18, -0.47
-22.91, 8.39
9.01, 24.98
-7.43, -45.0
31.43, 26.66
-15.38, -34.03
42.12, 48.28
13.27, 26.74
-39.07, -39.08
-47.22, 41.82
33.02, -38.56
-22.03, 10.61
44.16, -40.96
-12.67, 21.48
-2.5, 15.83
-43.04, 28.69
-22.99, -5.13
-15.77, 19.45
-6.21, 10.24
-2.81, -4.26
13.31, -39.67
48.51, 10.46
-24.38, -8.01
-30.89, 6.43
40.19, -33.51
-18.7, 8.32
46.93, 31.13
-33.5, 47.9
28.73, 25.63
20.71, 5.91
-23.83, -26.55
8.44, 19.88
5.71, -20.97
-5.43, -20.11
5.13, 47.76
-12.48, 49.71
4.07, -31.71
34.89, -16.36
-15.36, -28.46
-23.69, -17.7
-42.44, -31.0
29.93, 8.49
-6.03, 46.9
-3.22, 25.81
-35.85, 28.54
2.62, -20.63
18.3, -13.96
-42.78, 39.2
3.03, 33.57
-1.19, -28.01
-15.32, -3.15
5.14, 2.64
-5.97, -45.08
9.0, -13.72
7.07, -26.98
-11.12, -32.51
-38.7, -47.6
46.17, -48.4
48.72, -17.14
14.18, 10.4
-24.1, 0.01
25.05, -7.68
34.77, -41.06
-21.24, -45.82
-39.17, -10.9
-40.03, -41.5
-48.56, 48.21
-3.83, 38.83
-47.66, -46.4
-26.1, -20.3
44.03, 10.44
-40.09, 21.45
-49.51, 14.77
-0.87, 21.52
39.58, 8.44
8.3, -2.45
-25.55, -3.67
29.6, 45.88
36.0, -47.57
-0.65, 5.76
-21.81, 1.53
10.81, 41.54
-15.86, 28.91
0.62, 48.8
-21.42, -29.21
25.28, 33.51
-10.14, -46.84
-18.1, 35.68
12.1, 20.84
39.95, -40.69
18.63, -49.58
39.03, 14.91
11.52, -18.72
32.7, -36.21
-19.06, 10.18
9.8, -27.95
37.84, -19.72
15.99, -42.42
-15.67, 7.39
39.45, 16.8
-26.17, -5.58
-1.51, -49.44
5.43, 17.43
44.16, -35.5
6.71, 8.45
-23.14, -36.48
-20.94, 40.8
-4.94, -39.52
-27.39, -12.25
23.81, 0.95
-38.03, 27.94
25.3, 29.23
-38.96, 25.46
-36.07, -14.27
-46.36, -35.76
-39.01, 12.37
-3.61, -39.46
17.25, -3.3
47.94, -24.09
-0.5, 23.88
-25.63, -1.4
-32.84, 29.67
40.53, -35.39
9.57, 47.19
12.69, -13.52
10.91, 18.12
4.96, -18.45
36.34, 0.88
42.57, 10.19
5.78, -37.56
18.7, -24.0
21.29, 43.24
-13.99, -30.59
-31.99, 48.06
-46.21, -27.04
9.09, 23.92
-31.42, -43.35
44.06, 37.65
4.13, -6.23
36.34, -40.27
-5.82, 16.57
-2.26, -47.13
-12.92, 24.49
47.84, 33.01
-4.69, 49.1
-45.59, -24.23
42.83, 39.49
-6.71, -49.79
-48.37, -35.19
30.0, -15.42
-37.0, -44.33
-42.02, 21.97
27.71, -6.32
-42.07, -25.74
48.08, -39.05
21.42, -29.86
-36.69, 47.01
-35.94, -36.76
1.92, 4.29
-48.89, -39.84
-48.86, 6.62
-13.57, 34.53
-2.23, 6.28
1.33, -17.13
31.68, -44.78
-22.08, 46.6
-12.7, -0.37
1.7, -26.39
33.95, -6.14
3.59, 49.72
-44.16, 11.79
28.2, -13.32
3.86, 18.0
-49.49, 15.84
39.76, 11.77
-20.64, -42.29
17.1, 7.62
-21.94, -13.94
-6.17, -4.69
-41.06, -14.32
6.19, 37.14
40.25, 22.6
41.1, -14.26
26.1, -9.25
46.99, -23.58
27.31, 14.7
2.13, 22.84
22.65, -1.15
7.45, -39.63
-37.1, -48.29
-32.14, 12.21
35.34, -42.52
-9.59, -2.92
24.33, -1.26
-7.05, 22.35
17.77, 0.94
-49.4, -35.39
-40.97, -8.01
47.66, 42.62
-43.09, -37.8
25.18, -42.22
-11.49, -2.36
-39.91, -36.0
4.28, -16.99
2.06, -1.9
31.2, 48.47
9.19, 29.52
-49.7, -2.19
-41.0, 28.88
38.51, -2.72
34.43, 37.0
-47.33, 13.36
-7.01, 35.64
-8.96, 44.82
15.0, 43.25
-30.77, -40.11
-29.88, 37.0
9.12, 6.59
-35.76, 47.96
49.07, 17.6
45.12, 4.19
43.51, -37.35
48.53, -35.37
-21.11, 37.9
-2.85, -0.89
-33.62, 26.6
39.73, -17.24
42.55, 15.46
-26.18, 41.53
13.89, -34.4
-11.92, 13.34
48.78, -2.4
24.69, 35.84
-18.48, -26.49
27.07, 39.66
-48.16, 16.51
-48.25, 37.33
12.07, -20.56
7.02, -1.58
-38.67, 22.57
42.94, -28.8
32.23, -4.8
8.13, -45.2
9.25, 39.7
-20.37, 13.96
-46.97, -20.65
41.49, 42.39
41.42, 45.45
45.48, -45.2
-7.83, -30.77
24.04, 38.6
5.42, 44.15
37.33, -11.2
28.78, -8.12
-16.67, 14.83
-39.97, 39.61
47.82, 39.59
-28.74, -9.62
17.89, 42.21
-0.61, -26.23
40.9, -46.89
-47.41, -31.5
-32.22, 7.07
31.71, -15.76
-19.25, -22.27
-0.55, -38.27
-3.22, -30.6
-37.91, 35.48
4.71, -33.96
-30.83, 31.07
-36.61, 42.3
15.91, 27.46
-5.09, 4.92
23.28, 2.92
5.21, -36.27
-42.57, -16.32
19.29, 30.86
-34.72, -5.86
35.35, 19.19
16.87, 29.72
-38.38, -7.85
22.39, 13.17
8.44, -40.38
-34.4, 24.65
-27.55, -34.86
-7.04, 16.35
-31.27, -21.59
38.77, 6.64
38.04, -44.5
6.46, 28.26
5.07, 41.46
45.45, 18.51
4.13, 43.73
26.7, -32.34
-2.9, -23.21
23.59, 32.66
-10.83, -41.06
1.07, 39.66
-22.8, 49.27
-6.22, 41.1
3.87, -30.69
37.45, 22.29
-18.13, -24.03
-27.83, 33.87
-25.15, 3.65
13.08, 19.37
28.64, 19.64
-22.22, -25.22
49.35, -46.44
-18.1, -0.54
-38.84, 8.36
31.65, 33.11
10.3, 6.51
-39.46, -8.53
-35.73, -5.28
17.03, -43.97
-42.94, -36.37
35.79, -0.7
-45.43, 10.31
-43.42, 3.05
13.58, -22.86
-16.5, 37.03
1.88, 0.49
20.95, 29.6
-8.52, 46.49
11.09, -13.5
38.26, 37.05
42.94, 2.49
6.22, -0.65
-47.77, -48.85
-1.08, -3.61
-38.41, 0.08
-32.0, -15.9
14.15, -43.53
-28.05, 19.79
2.53, -17.84
29.83, 21.84
11.86, -41.09
23.34, -15.47
-20.09, -4.46
-33.68, 12.36
-33.29, 41.45
-31.07, -26.99
33.28, -12.39
44.36, 6.75
-41.04, 7.35
15.42, -18.23
-9.95, -5.54
34.06, -4.11
38.81, -6.12
-16.53, -26.16
26.99, -5.01
35.43, 18.1
-24.16, 37.21
-40.74, -1.46
38.46, -49.3
2.25, -27.43
22.17, 47.69
11.24, -0.9
48.93, 13.25
14.47, 44.17
-9.82, -20.25
29.11, -39.69
15.05, 8.23
35.49, -9.81
-26.31, -12.16
-5.45, -42.66
43.61, -47.42
7.8, -35.0
-19.43, 6.68
14.43, 46.31
44.28, 3.61
21.59, 9.75
-7.65, -48.73
13.05, 32.05
-26.18, -37.42
16.29, 24.54
46.51, 1.95
-29.22, -43.06
31.43, 12.15
10.77, -38.29
6.58, -9.77
-39.08, 29.57
12.54, 16.18
-19.63, -10.32
-46.69, 11.43
44.01, -28.83
-5.27, -47.9
37.15, 18.69
-7.51, -33.55
25.45, -0.54
-38.78, 2.93
-3.12, -38.58
-29.55, -29.77
15.95, -14.61
-36.31, 26.91
25.93, -40.68
-44.31, -25.33
39.99, 14.59
-5.88, 5.95
32.11, -47.67
30.71, -20.94
40.94, 43.39
-37.64, -45.83
-8.16, -17.85
29.94, 25.27
27.17, -49.71
-40.61, -12.66
14.85, -3.13
20.91, -10.09
43.45, -15.26
-10.73, 20.77
8.17, 32.41
41.37, -28.32
19.81, -7.81
-20.66, 35.02
-17.68, 0.78
34.76, 16.91
7.71, 45.51
-4.97, -28.36
14.41, -29.93
-44.56, 42.32
-10.01, -18.99
-26.34, 20.55
-20.04, 3.41
-29.83, 20.03
36.28, 49.82
-38.5, 17.25
2.97, 9.54
10.76, 0.25
32.67, 21.93
-26.48, 0.61
12.04, -48.59
33.13, -10.3
-46.4, 31.22
32.73, -21.68
-13.86, -46.76
42.23, -12.8
-48.74, -3.68
-18.45, -10.5
42.59, 30.24
-27.18, 25.8
11.48, -43.71
45.4, 31.69
-34.56, -33.4
-39.0, 45.38
-4.41, 1.38
22.15, 36.28
-36.34, -33.03
-40.36, -38.34
-25.64, 8.25
27.41, 44.87
-25.34, -39.28
-15.92, 10.13
-11.94, 48.09
47.58, -7.56
-23.58, -32.01
28.6, -48.54
35.95, 12.86
-3.92, -46.65
-28.67, 24.24
34.56, -30.75
36.25, -17.06
-17.14, 32.4
-9.37, 25.06
-9.05, -16.73
14.77, -11.88
31.38, -42.27
18.2, 36.41
-46.25, -46.15
15.93, -23.03
46.07, -32.23
-2.29, -40.13
-37.55, 41.85
-37.92, 49.36
11.89, -45.46
-35.39, -43.6
34.95, 1.2
-30.85, 29.91
23.46, 16.39
-12.71, -28.94
47.42, -34.24
35.43, 25.46
-2.31, 1.56
-40.92, 34.86
-14.14, -13.37
7.07, 46.62
-42.73, 15.02
-12.34, 26.5
-42.07, -32.27
-33.29, -45.77
-44.18, 40.34
3.58, -16.1
-30.34, 14.7
23.44, 12.42
34.03, 27.48
-43.66, -23.41
48.35, 36.23
12.86, 7.82
-0.03, 30.65
10.62, 32.29
-34.44, 39.28
34.09, 33.66
40.12, -9.87
-4.2, 34.01
-14.37, -1.13
-2.82, 2.93
-1.38, 32.98
24.41, -47.85
4.95, 25.04
49.74, -38.55
10.32, 37.3
21.81, -26.5
37.62, 42.84
21.71, -41.47
-37.78, -11.59
17.71, 22.04
28.6, -9.51
5.62, -2.65
20.01, 38.54
-10.8, -37.29
36.22, -45.99
-30.4, -44.39
-19.69, 23.18
-28.23, 11.46
31.68, 38.7
-19.73, -28.42
-29.22, 14.04
-32.45, 21.36
-19.76, 38.76
-13.89, -14.81
-35.01, 14.4
45.31, 36.94
-7.84, -7.8
-1.43, -12.76
-30.09, -37.58
-45.61, -14.01
8.22, -11.52
9.66, -21.11
6.06, 38.39
-7.63, -9.34
-3.9, -19.63
-38.49, -24.48
-27.77, 12.49
-14.93, -16.27
-15.06, 44.89
16.31, 8.35
22.44, 19.04
-2.7, 19.29
-22.22, 24.43
8.11, -21.96
25.23, -30.96
44.87, -21.97
-49.29, -28.55
27.96, -28.01
-1.61, -22.22
-23.6, -16.01
36.31, 40.35
30.84, 44.16
17.09, -1.03
-43.21, 35.96
2.01, 11.59
12.63, -33.37
-13.2, 19.59
-17.83, 22.71
-28.31, 2.91
-40.55, 13.26
-37.54, -42.81
24.86, -19.88
38.42, -10.49