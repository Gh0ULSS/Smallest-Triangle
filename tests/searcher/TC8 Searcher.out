read 1000 points
-30.14, -27.36
-31.07, -26.99
-32.06, -28.49
This is a triangle